apps/web/src/components/widget/InAppWidgetPreview.tsx

```diff
@@ -42,6 +44,83 @@ export function InAppWidgetPreview({
     onChange({});
   }
 
+  const editableContent = (
+    <ButtonsTemplatesPopover
+      isVisible={isButtonsTemplateVisible}
+      setIsPopoverVisible={setIsButtonsTemplateVisible}
+      setTemplateSelected={setIsButtonsTemplateSelected}
+      value={value}
+      onChange={onChange}
+    >
+      <Container
+        fluid
+        sx={{
+          position: 'relative',
+          padding: '15px 25px 0px',
+          borderRadius: '7px',
+          backgroundColor: theme.colorScheme === 'dark' ? colors.B20 : colors.white,
+          boxShadow: theme.colorScheme === 'dark' ? shadows.dark : shadows.medium,
+          width: preview ? undefined : '100%',
+
+          '&:before': {
+            content: '""',
+            position: 'absolute',
+            left: '0',
+            top: '0',
+            right: '0',
+            bottom: '0',
+            width: '5px',
+            borderRadius: ' 7px 0 0 7px',
+            background: colors.vertical,
+          },
```

Comment:
Is it worth to use Mantine's `createStyles` for this?

Reply:
yeah this just refactoring :)